installation/SETUP_INSTRUCTIONS.md

```diff
@@ -104,15 +104,23 @@ conda config --env --add channels robostack
 conda config --env --set channel_priority strict
 ```
 
-### Setup bd_spot_wrapper
+### Setup Perception and Utils package
+
+```bash
+# Generate module
+cd perception_and_utils/
+pip install -e . && cd ../
+```
+
```

Comment:
@akshararai @rutadesai @jimmytyyang @tusharsangam Do any of you know how to include this package in requirements for spot_rl and aria_data_utils through `setup.py`? These instructions are good but the more we could push installs to less number of `pip install` commands, the better.

Reply:
Let's address this in a follow-up BE task.